src/main/java/AlexLink/HomeWork/HW1/Task2.java

```diff
@@ -0,0 +1,10 @@
+package AlexLink.HomeWork.HW1;
```

Comment:
In this task you need:
1) Ask user for it's name
2) Print text with user's name